ft/ft/functions.py

```diff
@@ -376,3 +376,14 @@ def less_than(i, inp):
 def less_equals(i, inp):
     i = dynamic_cast(T_INT, i).value
     return inp <= i
+
+
+@register("format")
+@typed(None, T_STRING)
+def format(format_string, inp):
+    try:
+        return (format_string.value).format(float(inp))
+    except:
+        panic("Incorrect format string {} for input {}".format(
```

Comment:
Maybe add single quotes around the string and input (`'{}'`)?

Reply:
Good, yes!

Reply:
Ah, sorry.. I meant `"Incorrect format string '{}' for input '{}'"` :smile: 

I'll update this myself.